redfish_utilities/accounts.py

```diff
@@ -222,7 +225,19 @@ def get_user( context, user_name ):
     """
 
     avail_users = []
-    account_col = context.get( get_account_collection( context ) )
+
+    # if Password Change Required and user is same as login user, then redirect to Password Change Required's url
+    try:
```

Comment:
I don't think having this change here works very well; we lose sight of the user performing the request, and also have to trickle down all of the discover logic. I'll point this out a bit in rf_accounts.py.